diff --git a/src/nautilus-tag-manager.c b/src/nautilus-tag-manager.c
index fbd8571e5..6a94a016f 100644
--- a/src/nautilus-tag-manager.c
+++ b/src/nautilus-tag-manager.c
@@ -811,7 +811,7 @@ child_watch_cb (GPid     pid,
 static void
 export_tracker2_data (NautilusTagManager *self)
 {
-    gchar *argv[] = {"tracker3", "export", "--2to3", "files-starred", "--keyfile", NULL};
+    gchar *argv[] = {"@tracker3@/bin/tracker3", "export", "--2to3", "files-starred", "--keyfile", NULL};
     gint stdout_fd;
     GPid child_pid;
     g_autoptr (GError) error = NULL;
